.circleci/config.yml

```diff
@@ -7,6 +7,9 @@ jobs:
     docker:
       - image: circleci/ruby:2.6.3-node-browsers
       - image: circleci/postgres:9.4
+        environment:
```

Comment:
I think this should be indented with two more spaces (same for the two lines below)

Reply:
🤔 I'm seeing this is already on master without extra spaces, so wdyt?

Reply:
I was saying because of this
![image](https://user-images.githubusercontent.com/6373536/75989056-51b1bd00-5ed1-11ea-9f9c-e52f835437ad.png)

If there are other places that are incorrectly indented we could fix them too

Reply:
@santib `environment` was for `docker` block so I moved it back, moving it was generating an error in the schema check